assets/js/blocks/active-filters/edit.js

```diff
@@ -2,20 +2,30 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { InspectorControls } from '@wordpress/block-editor';
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
 import { Disabled, PanelBody, withSpokenMessages } from '@wordpress/components';
 import HeadingToolbar from '@woocommerce/editor-components/heading-toolbar';
 import BlockTitle from '@woocommerce/editor-components/block-title';
 import ToggleButtonControl from '@woocommerce/editor-components/toggle-button-control';
+import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Block from './block.js';
+import { getTextColorClassAndStyleFromAttributeObject } from '../../utils/style-attributes';
 
 const Edit = ( { attributes, setAttributes } ) => {
 	const { className, displayStyle, heading, headingLevel } = attributes;
 
+	const blockProps = useBlockProps( {
+		className,
+	} );
+
+	const textColorClassAndStyle = getTextColorClassAndStyleFromAttributeObject(
+		attributes
+	);
```

Comment:
Same here, I think we can remove those two variables and all styles will be applied by default.